spec/paperclip/storage/s3_spec.rb

```diff
@@ -342,18 +350,19 @@ def counter
 
   context "An attachment that uses S3 for storage and has spaces in file name" do
     before do
-      rebuild_model styles: { large: ['500x500#', :jpg] },
+      rebuild_model(
+        styles: { large: ['500x500#', :jpg] },
         storage: :s3,
         bucket: "bucket",
-        s3_credentials: {
-          'access_key_id' => "12345",
-          'secret_access_key' => "54321"
-        }
+        s3_credentials: { 'access_key_id' => "12345",
+                          'secret_access_key' => "54321"}
+        )
 
-        File.open(fixture_file('spaced file.png'), 'rb') do |file|
-          @dummy = Dummy.new
-          @dummy.avatar = file
-        end
+      File.open(fixture_file('spaced file.png'), 'rb') do |file|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.